### PR TITLE
[PR-8973] Updated docs for Shares API

### PR DESCRIPTION
**Pull request by human-spider**
## Demo

http://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-8973)

### CODE REVIEW

**Comment by dgtv** [comment]
@human-spider add Jira task to name and description, please

**Comment by human-spider** [comment]
@dgtv done. I don't think we need to bother Natalie with this one, the wording used is very simple.

**Comment by dgtv** [comment]
@human-spider AFAIK it is a part of our process for public documentation, so lets follow it

File: source/api_v2/shares.rst [hunk 1]
```diff
@@ -89,38 +90,48 @@ Create an Email share
...
-           "email": "friend2@example.com",
-           "type": "EmailOfferShare",
-           "short_url": "https://www.talkable.com/x/VsaTEe"
-         }
-       ]
-     }
-   }
+  {
+    "ok": true,
+    "reward": null,
```
Comment (dgtv) [inline source/api_v2/shares.rst]: `reward` should be under `result`. And lets add an `api_payload` for reward to one of examples.

File: source/api_v2/shares.rst [hunk 2]
```diff
@@ -23,32 +23,33 @@ Creates offer share.
                      Talkable dashboard after you log in and create a site.
    short_url_code    Offer short code obtained with
                      :ref:`origin creation <api_v2/origins>`.
+   channel           Options: `facebook`, `twitter`, `linkedin`, `whatsapp`,
```
Comment (dgtv) [inline source/api_v2/shares.rst]: I think `sms` should be added here as well

File: source/api_v2/shares.rst [hunk 3]
```diff
@@ -89,38 +99,48 @@ Create an Email share
...
+          "email_valid": true,
+          "self_referral": false,
+          "unsubscribed": false,
+          "blacklisted": false,
+          "meets_criteria": true,
+          "sharable": true
+        }
+      }
+    },
+    "reward": null
```
Comment (dgtv) [inline source/api_v2/shares.rst]: Please move this under `result`=)

File: source/api_v2/shares.rst [hunk 4]
```diff
@@ -89,38 +99,48 @@ Create an Email share
...
+          "currently_sent": true,
+          "previously_sent": false,
+          "email_valid": true,
+          "self_referral": false,
+          "unsubscribed": false,
+          "blacklisted": false,
+          "meets_criteria": true,
+          "sharable": true
+        }
+      }
```
Comment (dgtv) [inline source/api_v2/shares.rst]: `,` missed and remove it on the next line: `"reward": null,` -> `"reward": null`

Comment (human-spider) [reply source/api_v2/shares.rst]: 🤦‍♂️

**Pull request merged**